hazelcast/src/test/java/com/hazelcast/jet/core/ExecutionLifecycleTest.java

```diff
@@ -928,8 +1032,12 @@ private Job runJobExpectFailure(@Nonnull DAG dag, boolean snapshotting) {
             fail("Job execution should have failed");
         } catch (Exception actual) {
             String causeString = peel(actual).toString();
-            if (causeString == null
-                    || !(causeString.contains(MOCK_ERROR.toString()) || causeString.contains(CancellationException.class.getName()))) {
+            if (causeString == null  ||
+                !(causeString.contains(MOCK_ERROR.toString()) ||
+                         causeString.contains(CancellationException.class.getName()) ||
+                         causeString.contains(JetException.class.getName()) ||
+                        causeString.contains(NonSerializableException.class.getName()))
+            ) {
```

Comment:
I would expect something like this:
```suggestion
            if (causeString == null ||
                    !(causeString.contains(MOCK_ERROR.toString()) ||
                            causeString.contains(NON_SERIALIZABLE_EXCEPTION.toString()) ||
                            causeString.contains(CancellationException.class.getName()))
            ) {
```

Reply:
It changed into

`assertOneOfExceptionsInCauses(MockPMS.receivedCloseError.get(),
                NON_SERIALIZABLE_EXCEPTION,
                JET_EXCEPTION,
                SERIALIZATION_EXCEPTION,
                MOCK_ERROR,
                new CancellationException(),
                new JobTerminateRequestedException(CANCEL_FORCEFUL));`